Sources_v3/Models/CurrentUser.swift

```diff
@@ -49,6 +49,9 @@ public class _CurrentChatUser<ExtraData: UserExtraData>: _ChatUser<ExtraData> {
     /// A set of users flagged by the user.
     public let flaggedUsers: Set<_ChatUser<ExtraData>>
     
+    /// A set of message ids flagged by the user.
+    public let flaggedMessageIDs: Set<MessageId>
+    
```

Comment:
I just noticed: we have flagged users but flagged message ids. It should probably be `flaggedUserIds`, too. Also, muted user ids.

Reply:
We should mention here that these values are not synced too

Reply:
fixed in 9ac76c106a4f3d6dac95b55c2ae1209e9ace4e20 for both `flaggedUsers` and `flaggedMessageIDs`